scripts/semconv/templates/semantic_attributes.j2

```diff
@@ -19,344 +21,29 @@
 {%- endmacro %}
 
 from enum import Enum
-{%- if class == "SpanAttributes" %}
 
-from deprecated import deprecated
+class {{ root_namespace | to_camelcase(True) }}Attributes:
+  {%- for attribute in filtered_attributes %}
 
-{%- endif %}
-
-
-class {{class}}:
-    SCHEMA_URL = "{{schemaUrl}}"
-    """
-    The URL of the OpenTelemetry schema for these keys and values.
-    """
-    {%- for attribute in attributes | unique(attribute="fqn") %}
+    {% if attribute | is_template %}
+    {{attribute.fqn | to_const_name}}_TEMPLATE = "{{attribute.fqn}}"
+    {% else %}
     {{attribute.fqn | to_const_name}} = "{{attribute.fqn}}"
+    {% endif %}
     """
     {{attribute.brief | to_doc_brief}}.
 
     {%- if attribute.note %}
     Note: {{attribute.note | to_doc_brief | indent}}.
     {%- endif %}
 
-    {%- if attribute.deprecated %}
+    {%- if attribute | is_deprecated %}
     Deprecated: {{attribute.deprecated | to_doc_brief}}.
     {%- endif %}
     """
-{# Extra line #}
-    {%- endfor %}
-
-    {%- if class == "SpanAttributes" %}
-    # Manually defined deprecated attributes
```

Comment:
Curious as to why we are removing the deprecated attributes?

Reply:
we're bnot removing them from the repo, they are still here in the old location https://github.com/open-telemetry/opentelemetry-python/blob/9502a9b436a3e3566ee3a8a419ee56f2a6c80ba8/opentelemetry-semantic-conventions/src/opentelemetry/semconv/trace/__init__.py

but we're removing them from new locations and the new template - we don't want to carry them into the future.